test/lint/lint-assertions.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check for assertions with obvious side effects.
+
+import sys
+
+from subprocess import check_output, CalledProcessError
```

Comment:
nit: the code will more clearer if we import subprocess and then make call using `subprocess.check_output`.